homeassistant/components/hue/v2/binary_sensor.py

```diff
@@ -149,7 +154,36 @@ def is_on(self) -> bool | None:
 
 
 # pylint: disable-next=hass-enforce-class-module
-class HueGroupedMotionSensor(HueMotionSensor):
+class HueAreaMotionSensor(HueBaseEntity, BinarySensorEntity):
+    """Representation of a Hue area-based motion sensor."""
+
+    controller: (
+        GroupedMotionController
+        | SecurityAreaMotionController
+        | ConvenienceAreaMotionController
+    )
+    resource: GroupedMotion | SecurityAreaMotion | ConvenienceAreaMotion
+
+    entity_description = BinarySensorEntityDescription(
+        key="motion_sensor",
+        device_class=BinarySensorDeviceClass.MOTION,
+        has_entity_name=True,
+    )
+
+    @property
+    def is_on(self) -> bool | None:
+        """Return true if the binary sensor is on."""
+        if getattr(self.resource, "enabled", True) is False:
+            return False
+        if not (motion_feature := self.resource.motion):
+            return None
+        if motion_feature.motion_report is not None:
+            return motion_feature.motion_report.motion
+        return motion_feature.motion
```

Comment:
the solution is just copying this block of code into the "is_on" property of the HueMotionSensor